receiver/mongodbatlasreceiver/metadata.yaml

```diff
@@ -579,15 +579,15 @@ metrics:
     description: Disk partition space
     extended_documentation: Aggregate of MongoDB Metrics DISK_PARTITION_SPACE_FREE, DISK_PARTITION_SPACE_USED
     unit: By
-    attributes: [memory_state]
+    attributes: [disk_status]
```

Comment:
Why do we need to repeat "disk"? "status" is not enough?

As example for host metrics we don't have "memory_state" and "cpu_state".